docs/docs/concepts/foundation/communication/public_private_calls/slow_updates_tree.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Privately access Public State
+---
+
+In Aztec, private and public execution environments are completely separate and operate with distinct state management. In the previous section we learned that functions can call private functions, and public functions can save messages to a merkle tree to then be read by a private function.
+
+It is also possible to access persistent data across both private and public state using a **slow updates tree**, which we will talk about on this page. Please note that we are still experimenting with this feature and it is not something that has ever been achieved before.
+
+On this page you will learn:
+
+1. Why a slow updates tree exists & use cases
+2. How it works
+3. How it can be used to access data in public & private domains
+4. Limitations
+
+## The need for a slow updates tree
```

Comment:
really like the idea behind this section! But need to fix the "access data across private and public" connotation

Reply:
fixed